docs/en/install-upgrade/air-gapped-install.asciidoc

```diff
@@ -0,0 +1,532 @@
+  local url_suffix="$1"
+  local package_prefixes="$2"
+
+  local _url_suffixes="$url_suffix ${url_suffix}.sha512 ${url_suffix}.asc"
+  local _pkg_dir=""
+  local _dl_url=""
+
+  for _download_prefix in $package_prefixes; do
+    for _pkg_url_suffix in $_url_suffixes; do
+          _pkg_dir=$(dirname ${DOWNLOAD_BASE_DIR}/${_download_prefix})
+          _dl_url="${ARTIFACT_DOWNLOADS_BASE_URL}/${_download_prefix}-${_pkg_url_suffix}"
+          (mkdir -p $_pkg_dir && cd $_pkg_dir && curl -O "$_dl_url")
+    done
+  done
+}
+
+# and we download
+for _os in linux windows; do
+  case "$_os" in
+    linux)
+      PKG_URL_SUFFIX="${STACK_VERSION}-${_os}-x86_64.tar.gz"
+      ;;
+    windows)
+      PKG_URL_SUFFIX="${STACK_VERSION}-${_os}-x86_64.zip"
+      ;;
+    *)
+      echo "[ERROR] Something happened"
+      exit 1
+      ;;
+  esac
+
+  download_packages "$PKG_URL_SUFFIX" "$COMMON_PACKAGE_PREFIXES"
+  
+  if [[ "$_os" = "windows" ]]; then
+    download_packages "$PKG_URL_SUFFIX" "$WIN_ONLY_PACKAGE_PREFIXES"
+  fi
+
+  if [[ "$_os" = "linux" ]]; then
+    download_packages "${STACK_VERSION}-x86_64.rpm" "$RPM_PACKAGES"
+    download_packages "${STACK_VERSION}-amd64.deb" "$DEB_PACKAGES"
+  fi
+done
+
+
+## selinux tweaks
+# semanage fcontext -a -t "httpd_sys_content_t" '/opt/elastic-packages(/.*)?'
+# restorecon -Rv /opt/elastic-packages
+
+----
+
+The following is an example NGINX configuration for running a web server for the {artifact-registry}.
+
+[source,shell]
+----
+user  nginx;
+worker_processes  2;
+
+error_log  /var/log/nginx/error.log notice;
+pid        /var/run/nginx.pid;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log          /var/log/nginx/access.log  main;
+    sendfile            on;
+    keepalive_timeout   65;
+
+    server {
+        listen                  9080 default_server;
+        server_name             _;
+        root                    /opt/elastic-packages;
+
+        location / {
+
+        }
+    }
+
+}
+
+----
+
+[discrete]
+[[air-gapped-epr-kubernetes-example]]
+=== Appendix C - EPR Kubernetes Deployment
+
+The following is a sample EPR Kubernetes deployment YAML file.
+
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: elastic-package-registry
+  namespace: default
+  labels:
+    app: elastic-package-registry
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: elastic-package-registry
+  template:
+    metadata:
+      name: elastic-package-registry
+      labels:
+        app: elastic-package-registry
+    spec:
+      containers:
+        - name: epr
+          image: docker.elastic.co/package-registry/distribution:8.6.1
```

Comment:
Fixed version number

Reply:
Fixed in [40dbeb3](https://github.com/elastic/stack-docs/pull/2458/commits/40dbeb3c10d7aa1d5314b996b9c13f4a0acf5e9c)